js/localization/messages/hu.json

```
@@ -305,12 +305,93 @@
     "dxFileManager-errorNoAccess": "Hozzáférés megtagadva. A műveletet nem lehet befejezni.",
     "dxFileManager-errorDirectoryExistsFormat": "'{0}' nevű könyvtár már létezik.",
     "dxFileManager-errorFileExistsFormat": "'{0}' nevű fájl már létezik.",
-    "dxFileManager-errorFileNotFoundFormat": "'{0}' nevű fájl nem található",
-    "dxFileManager-errorDirectoryNotFoundFormat": "'{0}' nevű könyvtár nem található",
-    "dxFileManager-errorWrongFileExtension": "Nem engedélyezett fájltípus",
-    "dxFileManager-errorMaxFileSizeExceeded": "A fájlméret meghaladja a megengedett legnagyobb méretet",
+    "dxFileManager-errorFileNotFoundFormat": "'{0}' nevű fájl nem található.",
+    "dxFileManager-errorDirectoryNotFoundFormat": "'{0}' nevű könyvtár nem található.",
+    "dxFileManager-errorWrongFileExtension": "Nem engedélyezett fájltípus.",
+    "dxFileManager-errorMaxFileSizeExceeded": "A fájlméret meghaladja a megengedett legnagyobb méretet.",
+    "dxFileManager-errorInvalidSymbols": "A név érvénytelen karaktereket tartalmaz.",
     "dxFileManager-errorDefault": "Ismeretlen hiba.",
-    "dxFileManager-errorDirectoryOpenFailed": "TODO",
+    "dxFileManager-errorDirectoryOpenFailed": "A könyvtár nem nyitható meg",
+
+    "dxFileManager-commandCreate": "Új könyvtár",
+    "dxFileManager-commandRename": "Átnevezés",
+    "dxFileManager-commandMove": "Áthelyezés",
+    "dxFileManager-commandCopy": "Másolás",
+    "dxFileManager-commandDelete": "Törlés",
+    "dxFileManager-commandDownload": "Letöltés",
+    "dxFileManager-commandUpload": "Fájlok letöltése",
```

```suggestion
    "dxFileManager-commandUpload": "Feltöltés",
```

Should it be `Feltöltés` here?